### PR TITLE
Renamed Mexico City to Runtime Manager

### PR DESCRIPTION
**Pull request by dominic-mulligan-arm**
Documentation, comments, code, and build scripts.

### CODE REVIEW

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 769ad300a60dac496a26c1ce585e23759f8178d8
* Result: FAILED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=34cf16fd-10a5-4aa9-89e3-18014a7a82c0%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: sinaloa/src/sinaloa_tz.rs [hunk 1]
```diff
@@ -21,15 +21,19 @@ pub mod sinaloa_tz {
     };
     use std::convert::TryInto;
     use std::sync::Mutex;
-    use veracruz_utils::{EnclavePlatform, SgxRootEnclaveOpcode, MCOpcode, SGX_ROOT_ENCLAVE_UUID, MC_UUID};
+<<<<<<< HEAD
+    use veracruz_utils::{EnclavePlatform, SgxRootEnclaveOpcode, RuntimeManagerOpcode, SGX_ROOT_ENCLAVE_UUID, RUNTIME_MANAGER_UUID};
+=======
+    use veracruz_utils::{EnclavePlatform, JaliscoOpcode, RuntimeManagerOpcode, JALISCO_UUID, RUNTIME_MANAGER_UUID};
+>>>>>>> a46d55a (mexico-city-renaming: further renaming of Mexico City)
```
Comment (ShaleXIONG) [inline sinaloa/src/sinaloa_tz.rs:28]: I believe the top (HEAD) is what we want in term of this conflict.

File: sinaloa/src/sinaloa_tz.rs [hunk 2]
```diff
@@ -53,13 +57,17 @@ pub mod sinaloa_tz {
...
                     *ji_guard = true;
                 }
             }
 
-            let mc_uuid = Uuid::parse_str(&MC_UUID.to_string())?;
+<<<<<<< HEAD
+            let mc_uuid = Uuid::parse_str(&RUNTIME_MANAGER_UUID.to_string())?;
+=======
+            let runtime_manager_uuid = Uuid::parse_str(&RUNTIME_MANAGER_UUID.to_string())?;
+>>>>>>> a46d55a (mexico-city-renaming: further renaming of Mexico City)
```
Comment (ShaleXIONG) [inline sinaloa/src/sinaloa_tz.rs:70]: I believe the bottom (a46d55a) is what we want in term of this conflict.

**Comment by dominic-mulligan-arm** [comment]
@ShaleXIONG, thanks, I missed those conflicts during a rebase earlier.

**Review by ShaleXIONG — APPROVED** [review]

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 900a95d41d3081351342cc1f72186037bc8032fb
* Result: FAILED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=07b08b21-a80a-4a44-8d0d-86d19797be4a%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 2a9d2b13707730f7aa6d4270ba31438a2ad95ea6
* Result: FAILED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=d341ef6a-b498-4d86-8702-b1b4e330d582%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by gbryant-arm — APPROVED** [review]
LGTM

**Review by ShaleXIONG — DISMISSED** [review]

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: e11d90c741f94367747b3de0193acc71c2408557
* Result: SUCCEEDED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=249f7d78-e50e-492f-a68a-c8abe0459541%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by gbryant-arm — CHANGES_REQUESTED** [review]
I don't know if you're done rebasing.
fwiw some symbols have not been renamed: "mc", "mc_nitro", "MCMessage", "mch_guard" (non exhaustive)

**Comment by dominic-mulligan-arm** [comment]
> I don't know if you're done rebasing.
> fwiw some symbols have not been renamed: "mc", "mc_nitro", "MCMessage", "mch_guard" (non exhaustive)

Yeah, I seem to have messed up a recent rebase.

**Review by gbryant-arm — DISMISSED** [review]
LGTM

File: runtime-manager/Makefile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Makefile
+			# Makefile
```
Comment (ShaleXIONG) [inline runtime-manager/Makefile:1]: There is an unexpected white space here. Not a big deal though.

**Review by ShaleXIONG — APPROVED** [review]

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: d8c9bb0211867c08e352641d8436321aaa0000cc
* Result: FAILED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=ad777849-9b8a-4a1b-93aa-d12056bdad08%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: a63dc294c1354f2f2e640dc7aeeb72b9404583c4
* Result: FAILED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=c3539cca-b1e5-4271-a549-c30a0f37d962%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 64d88c4f3386cba06800adda519057ac335054bb
* Result: SUCCEEDED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=235c44ce-8aab-4a33-9586-773010eb227b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by dominic-mulligan-arm** [comment]
+1 reached, merging.

**Pull request merged**